english/plow/plow.txt

```diff
@@ -0,0 +1,6 @@
+Mac: Great, onto the matter at hand. We're getting plowed in the ass by the oil companies and the gas companies with their ten gallon hats and their rotten ass-plowing hearts. So, as the brains of this organization, I came up with a plan.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.